src/screens/SignInScreen.tsx

```diff
@@ -1,76 +1,94 @@
         label={t('common.email_label')}
         control={control}
         errors={errors}
+        name="email"
         keyboardType="email-address"
+        autoCapitalize="none"
+        onSubmitEditing={passwordInputRef.current?.focus}
         placeholder={t('common.email_placeholder')}
-        name="email"
-        type="email"
-        returnKeyType="next"
-        onSubmitEditing={emailInputRef.current?.focus}
         rules={{
           required: t('form.required'),
           pattern: {
             value: REGEX.EMAIL,
             message: t('form.invalid_email_format'),
           },
         }}
-        containerStyle={[s.mB2]}
       />
       <ControlledInput
-        ref={emailInputRef}
-        fullWidth
+        isRequired
+        ref={passwordInputRef}
+        returnKeyType="send"
+        onSubmitEditing={submit}
         label={t('sign_in_screen.password_label')}
         control={control}
         errors={errors}
-        placeholder={t('sign_in_screen.password_placeholder')}
         name="password"
-        type="protected"
-        returnKeyType="send"
-        onSubmitEditing={submit}
-        rules={{ required: t('form.required') }}
-        containerStyle={[s.mB2]}
-      />
-      <ControlledCheckbox
-        control={control}
-        name="confirm"
-        label="Remember me"
-        disabled={isSubmitting}
+        type="password"
+        autoCapitalize="none"
+        placeholder={t('sign_in_screen.password_placeholder')}
+        rules={{
+          required: t('form.required'),
+        }}
       />
-      <Box mt={4} />
-      <Button onPress={submit} title="Sign in" loading={isSubmitting} disabled={isSubmitting} />
-      <Box mt={8} />
-      {!!error && <Text.BodyBold color="errorLight">{error}</Text.BodyBold>}
+      <Center mt={8}>
+        <ControlledCheckbox
+          control={control}
+          errors={errors}
+          name="confirm"
+          label={t('sign_in_screen.remember_me')}
+          mb={4}
+        />
+        <Button
+          testID="signInButton"
+          onPress={submit}
+          isLoading={isSubmitting}
+          width="64"
+          isDisabled={isSubmitting}
+          mb={8}
+        >
+          {t('sign_in_screen.sign_in')}
+        </Button>
+        <Text bold mb={4}>
+          {t('sign_in_screen.dont_have_an_account')}
+        </Text>
+        <Button width="64" onPress={goToSignUp} variant="ghost">
+          {t('sign_in_screen.sign_up')}
+        </Button>
+      </Center>
+      <Box mt={12} />
       {/* TODO: Remove this after implementing signing in with backend  */}
-      <Box alignItems="center">
-        <Text.H5>{t('sign_in_screen.dont_have_an_account')}</Text.H5>
-        <Box mt={2} />
-        <Button onPress={goToSignUp} title="Sign up" variant="Flat" />
-      </Box>
-      <Box mt={8} />
-      <Text.BodyBold>Correct credentials</Text.BodyBold>
-      <Text.BodyRegular color="gray500" center>
+      <Text bold>Correct credentials</Text>
```

Comment:
Move to i18n

Reply:
I would suggest not moving that label to i18n as it seems to be temporary code, under the `TODO` section. Moving it to i18n would require from developer to keep in mind checking one more place in case of removal them.